src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin text-secondary() {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), font-size, line-height, and font-weight properties should be ordered alphabetically.